android/src/main/java/com/example/flutter_braintree/FlutterBraintreeCustom.java

```diff
@@ -128,8 +140,19 @@ protected void requestGooglePayNonce() {
         googlePayClient.setListener(this);
 
         mTotalPrice = intent.getStringExtra("totalPrice");
+        Log.d(TAG, "mTotalPrice = "+mTotalPrice);
+        if (mTotalPrice == null) return;
+
         String currencyCode = intent.getStringExtra("currencyCode");
+        Log.d(TAG, "currencyCode = "+currencyCode);
+        if (currencyCode == null) {
+            // default currency: HKD
+            currencyCode = "HKD";
```

Comment:
I do think we should assume / apply default currency as HKD.

Reply:
updated